LiskHQ/lisk-hub
# Add discreet toggle to the sign-in page - Closes #2332

### What issue have I solved?

Closes #2332 
Closes #2151

### How have I implemented/fixed it?

This PR contains the following implementations:

- [x] Create DiscreetModeToggle component.
- [x] Added into the Sign in page.
- [x] Remove the flags in settings component as the implementation is finished.
- [x] Unit test created for the DiscreetModeToggle component

**NOTE:**The discreet mode element was added in the Sign In page centered and between the passphrase input and footer buttons, as in the design show up as part of the passphrase element.
This was discussed with Dom and they are still working to see where will be the final design for all the arrange of elements in this page. So in summary they agree in have the discreet mode element where it is right now in this PR.

### How has this been tested?

1. Run the app.
2. Go to the Sign In page, enable the discreet mode
3. Sign in
4. All the values in the app should be blur (all values related to the owner account).
5. Go to setting and the Discreet mode option should be enabled
6. Then disabled the option
7. Check the values again in the app, everything should be visible.
8. Sign out.
9. Go to Sign in page again, discreet mode should be disabled.

Discreet Mode could be enable/disable in Sign In page and Settings page


### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)
